credsweeper/utils/util.py

```diff
@@ -372,6 +375,30 @@ def yaml_dump(obj: Any, file_path: str, encoding=DEFAULT_ENCODING) -> None:
         except Exception as exc:
             logging.error(f"Failed to write: {file_path} {exc}")
 
+    @staticmethod
+    def __extract_value_value(node: Any, value_value: Any) -> List[Any]:
```

Comment:
How about rename the method with `__extract_value()`?
IMO `value_value` seems not that good..